package/MDAnalysis/core/AtomGroup.py

```diff
@@ -546,7 +552,9 @@ def position(self):
 
         :Returns: a (3,) shape numpy array
         """
-        return self.universe.coord.positions[self.index]  # internal numbering starts at 0
+        return self.universe.coord.positions[
```

Comment:
this line change is only done because of the trailing comment. Converting this into a normal comment in the above line will make yapf behave better.